test/CMakeLists.txt

```diff
@@ -150,7 +149,6 @@ if(NRN_ENABLE_PYTHON AND PYTEST_FOUND)
     # Launching mpi executable with full path can mangle different python versions and libraries
```

Comment:
I think this comment, or something based on it, should move to the top-level `CMakeLists.txt` to explain **why** we are defining this `MPIEXEC_NAME` variable.

Reply:
I was hesitating, but you're right.